extensions/toggl-track/src/components/CreateTimeEntryForm.tsx

```diff
@@ -16,6 +16,7 @@ function CreateTimeEntryForm({ project, description }: { project?: Project; desc
     try {
       await toggl.createTimeEntry({
         projectId: selectedProject?.id,
+        workspaceId: selectedProject?.workspace_id || (me as Me).default_workspace_id,
```

Comment:
I'm not 100% sure about this cast. Is it possible that user can get here without `me` loaded correctly? If so, there will be an error thrown here. Should I verify it and use `showToast` instead?

Reply:
The `/me` request would have to fail for some weird reason, but it's always possible. I'd add a `showToast` just in case it does fail.

Reply:
This is already typed as `Me | null`. Change accordingly:
```suggestion
        workspaceId: selectedProject?.workspace_id || me?.default_workspace_id,
```

Reply:
The value can’t be undefined

Reply:
@bkeys818 it can before it's fetched